src/screens/ProfileScreen/elements/AnonymousTab.js

```diff
@@ -23,21 +23,21 @@ const styles = StyleSheet.create({
     fontSize: 14,
     lineHeight: 17,
     textAlign: 'center',
-    color: isActive ? COLORS.signed_primary : COLORS.blackgrey
+    color: isActive ? COLORS.anon_primary : COLORS.blackgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.signed_primary` to `COLORS.anon_primary` seems appropriate given the context of the `AnonymousTab` component. This change should make the active state more visually distinct for anonymous tabs.
```diff
-    color: isActive ? COLORS.signed_primary : COLORS.blackgrey
+    color: isActive ? COLORS.anon_primary : COLORS.blackgrey
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->